src/sentry/event_manager.py

```diff
@@ -1361,6 +1363,17 @@ def _nodestore_save_many(jobs: Sequence[Job]) -> None:
             if unprocessed is not None:
                 subkeys["unprocessed"] = unprocessed
 
+        if app_feature:
```

Comment:
Should this imply that app_feature could be None? If yes I think the type signature is wrong.

Reply:
app_feature would not be None. But I am being a bit cautious here since I do not know whether mypy can be trusted with this file.